09-Michael-A-Anderson/wk01/5-fri/MTA/js/script.js

```diff
@@ -0,0 +1,125 @@
+function stationFactory(line,name){
+    return {
+        name: name,
+        line:line
+    }
+}
+
+const N = [
+    stationFactory('N','8th'),
+    stationFactory('N','Union Square'),
+    stationFactory('N','23rd'),
+    stationFactory('N','28th'),
+    stationFactory('N','35th'),
+    stationFactory('N','Times Square')
+]
```

Comment:
```suggestion
const N = ['8th', 'Union Square', '23rd', '28th', '35th', 'Times Square'];
```
This is a simpler way to structure your data.